src/csagent/product/nodes.py

```
@@ -1,178 +1,94 @@
+
+    available_products = get_products()
+    if product not in available_products:
+        return f"Product '{product}' not found. Available products: {', '.join(available_products)}"
+
+    try:
+        products_dir = get_resources_dir() / "products"
+        with open(
+            products_dir / f"{product}_{language}.md", "r", encoding="utf-8"
+        ) as f:
+            return f.read()
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)